otter/convergence/selfheal.py

```diff
@@ -0,0 +1,193 @@
+"""
+Self heal service.
```

Comment:
It would be helpful to describe in a few brief sentences what the "self heal service" is.